src/main/java/org/hl7/davinci/priorauth/endpoint/ClaimEndpoint.java

```diff
@@ -209,6 +210,12 @@ private Bundle processBundle(Bundle bundle) {
     ClaimStatus status = claim.getStatus();
     Disposition responseDisposition = null;
     ClaimResponseStatus responseStatus = ClaimResponseStatus.ACTIVE;
+    if (FhirUtils.isClaimInquiry(bundle)) {
+
+      // handle the claim inquiry operation
+      Bundle responseBundle = ClaimResponseFactory.handleClaimInquiry(bundle);
+      return responseBundle;
+    }
```

Comment:
So the Claim Inquiry is it's own new operation at the endpoint `Claim/$inquiry`. This is part of the `Claim/$submit` workflow.